frontend-nx/apps/edu-hub/components/common/Dot.tsx

```diff
@@ -1,25 +1,30 @@
 import { FC } from 'react';
 import { GoPrimitiveDot } from 'react-icons/go';
+export type DotColor = 'grey' | 'lightgreen' | 'orange' | 'red' | string;
```

Comment:
It should not be possible to use the dots with any color.
That is the ` | string` at the end should be removed. That way the set of colors used is completely clear and can also be changed at any time for all dots by catching the strings and forwarding other colors within the component.